pyproject.toml

```diff
@@ -137,9 +137,9 @@ log_cli_level = "INFO"
 xfail_strict = true
 filterwarnings = [
     "error",
-    'ignore:.*`product` is deprecated.*:DeprecationWarning:qiskit:',
     'ignore:.*qiskit.__qiskit_version__.*:DeprecationWarning:qiskit:',
-    'ignore:.*encountered in det.*:RuntimeWarning:numpy.linalg:',
```

Comment:
Seems like you still need to keep this one in. Apple Silicon still has this issue as seen by the CI run.

Reply:
done